### PR TITLE
for #20: start onboarding UI in pop-up

### PR DESCRIPTION
**Pull request by groovecoder**
Work-in-progress; still needs:

* [ ] Final image assets
* [ ] Final copy
* [ ] What else? @johngruen @TanviHacks

testpilot-containers-onboard

### CODE REVIEW

File: webextension/js/popup.js [hunk 1]
```diff
@@ -40,6 +40,33 @@ function showContainerTabs(containerId) {
   hideorshowIcon.src = CONTAINER_HIDE_SRC;
 }
 
+if (localStorage.getItem('onboarded2')) {
```
Comment (groovecoder) [inline webextension/js/popup.js]: @jonathanKingston - I'm really not happy with this logic for controlling the shown/hidden state of the onboarding vs. container panels. I'll keep poking at this code to improve it.

**Comment by TanviHacks** [comment]
Panel that shows up when you press the arrow for a container.

Edit Containers section.

**Comment by groovecoder** [comment]
@TanviHacks do you mean each of those elements needs its own onboarding tutorial/panel?

**Comment by TanviHacks** [comment]
Sorry, my comment makes no sense for this context.  Please ignore it.

**Comment by groovecoder** [comment]
Deleted extra comments. Assigning to @jonathanKingston for review & spot-check.

Note: This depends on us having Firefox 52 ContextualIdentityService module, so we need to get that sorted.

**Comment by groovecoder** [comment]
Rebased this branch on @bakulf and @jonathanKingston's latest commits in master.

**Pull request merged**